File: Voronezh/Bars.Gkh.Regions.Voronezh/libs/B4/model/RosRegExtractReg.js
Ext.define('B4.model.RosRegExtractReg', {
    extend: 'B4.base.Model',
    idProperty: 'Id',

    proxy: {
        type: 'b4proxy',
        controllerName: 'RosRegExtractReg'
    },
    fields: [
        { name: 'Id' },
        { name: 'Reg_ID_Record' },
        { name: 'Reg_RegNumber' },
        { name: 'Reg_Type' },
        { name: 'Reg_Name' },
        { name: 'Reg_RegDate' },
        { name: 'Reg_ShareText' }
    ]
});

File: Voronezh/Bars.Gkh.Regions.Voronezh/Entities/RosRegExtract/RosRegExtractReg.cs
using Bars.B4.DataAccess;
using Bars.Gkh.Entities;

namespace Bars.Gkh.Regions.Voronezh.Entities
{
    public class RosRegExtractReg : PersistentObject
    {
        //Registration
        public virtual string Reg_ID_Record { get; set; }
        public virtual string Reg_RegNumber { get; set; }
        public virtual string Reg_Type { get; set; }
        public virtual string Reg_Name { get; set; }
        public virtual string Reg_RegDate { get; set; }
        public virtual string Reg_ShareText { get; set; }

    }
}
